src/basic/macro.h

```diff
@@ -9,6 +9,8 @@
 #include <sys/sysmacros.h>
 #include <sys/types.h>
 
+#include "macro-common.h"
```

Comment:
if we go for the "fundamental" moniker for the dir, this should probably be called "macro-fundamental.h" or so, i guess